### ISSUE TITLE
Build failed on windows

### ISSUE THREAD

**Issue by Aravindbolloju** [google/visqol#75]
Road to build the solution on windows is hard .. still couldn't figured it out. Does anyone succeeded in setting up this on windows?

**Comment by ipio** [comment]
I agree, I couldn't find a way to build it on Windows after following the steps they lay out. 

My workaround has been to run the following docker container running under Windows:
https://hub.docker.com/r/jonashaag/visqol

However, the documentation listed on that docker page a bit confusing. Here's what worked for me:

docker run -it -v c:/users/<your username>/visqol/data:/data jonashaag/visqol:v3 --degraded_file /data/degraded.wav --reference_file /data/reference.wav

Hope that helps!

**Comment by mchinen** [comment]
Sorry that this is causing issues.  Thank you for the solution @ipio .   It's true that we primarily develop on Linux, so our Windows support is not as good as it should be.  One approach that may make sense in the future is to use the CI actions to build releases for those that aren't interested in developing but just using ViSQOL as a binary.

**Comment by Aravindbolloju** [comment]
> Sorry that this is causing issues. Thank you for the solution @ipio . It's true that we primarily develop on Linux, so our Windows support is not as good as it should be. One approach that may make sense in the future is to use the CI actions to build releases for those that aren't interested in developing but just using ViSQOL as a binary

Hey, I didn't immediately posted it here... After I raised this issue I worked for its successful build on windows and finally it succeeded. Some changes I have done in the build files and then it succeeded ultimately.... I verified it with Linux build as well and the scores generated are intact. One issue I noticed is the windows one that I build is failed in case of speech mode but in audio mode it is going well and I'm using it.

**Comment by Hyeonji-Jung** [comment]
Hello, it's amazing that you succeeded in building in a window environment. I'm also trying to build in Windows.
However, memory problems and errors occur in the process of building tensorflow.
Is it the only solution to build the tensorflow directly into the bazel to use visqol?
I would really appreciate it if you could let me know through what modifications you made to build it successfully.
